.github/workflows/update-dependencies.yml

```diff
@@ -0,0 +1,31 @@
+name: Update python dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  "0 23 * * *"
+
+jobs:
+  update-dependencies:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v4
+    - uses: "opensafely-core/setup-action@v1"
+      with:
+        python-version: "3.8"
+        install-just: true 
+    
+    - uses: actions/create-github-app-token@v1
+      id: generate-token
+      with:
+        app-id: 1031449  # opensafely-core Create PR app
+        private-key: ${{ secrets.CREATE_PR_APP_PRIVATE_KEY }}
+
+    - uses: bennettoxford/update-dependencies-action@v1
+      with:
+        token: ${{ steps.generate-token.outputs.token }}
+        # note: we don't current auto update external production dependencies,
```

Comment:
```suggestion
        # note: we don't currently auto update external production dependencies,
```

Reply:
oh no, don't accept the suggestion, sorry

Reply:
lol!

Reply:
Have pushed the typo fix